PubNub/Data/PNAES.m

```diff
@@ -356,8 +359,12 @@ + (NSError *)errorFor:(CCCryptorStatus)status {
             errorCode = kPNAESInsufficientMemoryError;
             break;
         case kCCDecodeError:
+#ifdef PGDROID
+#   warning HC SVNT DRACONES! : CommonCrypto is not fully implemented on PGDroid ...
```

Comment:
Just a simple compiler warning that our security/crypto layer is severely lacking on Android.